editor/editor_node.cpp

```diff
@@ -7073,63 +7073,65 @@ EditorNode::EditorNode() {
 
 	// More visually meaningful to have this later.
 	raise_bottom_panel_item(AnimationPlayerEditor::get_singleton());
-	add_editor_plugin(memnew(ReplicationEditorPlugin));
 
 	add_editor_plugin(VersionControlEditorPlugin::get_singleton());
-	add_editor_plugin(memnew(ShaderEditorPlugin));
-	add_editor_plugin(memnew(ShaderFileEditorPlugin));
 
+	add_editor_plugin(memnew(AnimationTreeEditorPlugin));
```

Comment:
I'd add a comment to introduce the editor plugins block, make clear that it should be ordered alphabetically in the appropriate category.

My experience shows that a significant proportion of contributors don't infer "alphabetical ordering" when they see a clearly alphabetically ordered list.

Reply:
Done, added a comment.